db/src/main/kotlin/edu/artic/db/RetrofitAppDataServiceProvider.kt

```diff
@@ -22,6 +29,11 @@ class RetrofitAppDataServiceProvider(
         const val EVENT_HEADER_ID = "events_download_header_id"
     }
 
+    /**
+     * This is used to access the correct 'events' and 'exhibitions' urls.
+     *
+     * Similar concept in play as with [AppDataManager.dataObject].
+     */
     val dataObject: Subject<ArticDataObject> = BehaviorSubject.create()
```

Comment:
This should be removed and implementation should be replaced with `AppDataManager` class. I do not think this belongs here anymore then or at least make this `private` so other classes can't see it.

Reply:
By "[replace] with AppDataManager class" you're suggesting I inject `AppDataManager` then subscribe to the `AppDataManager.dataObject`, correct? That seems workable.

Reply:
possibly, but it'll lead to circular dependencies, which is not allowed since `AppDataManager` already depends on this class.

Reply:
What if I move our cannonical `dataObject` field into the DAO class? That would solve the dependency concern, but at a cost.